src/app/app.component.html

```diff
@@ -1,13 +1,12 @@
 <nav>
   <mat-toolbar>
-    <span class="brand">Prylabs</span>
-
-    <a routerLink="/" class="mat-button">Home</a>
-    <a routerLink="/stats" class="mat-button">Stats</a>
-    <a routerLink="/participate" class="mat-button">Participate</a>
+    <span class="brand">💎 Prysm Eth2 Testnet</span>
 
     <span class="flex-spacer"></span>
-    <span class="small-text" *ngIf="numValidators">{{numValidators}} validators!</span>
+    <a routerLink="/" mat-button>Home<mat-icon>home</mat-icon></a>
+    <a routerLink="/stats" mat-button>Stats<mat-icon>public</mat-icon></a>
+    <a routerLink="/participate" mat-button>Participate<mat-icon>person_add</mat-icon></a>
+    <a target="_blank" href="https://goerli.etherscan.io/address/{{depositContractAddress}}" *ngIf="numValidators" mat-button>{{numValidators}} validators!<mat-icon>launch</mat-icon></a>
```

Comment:
I don't think this looks very good as a button. It's meant to just be some additional/fun information.